code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -33,9 +33,10 @@
 			if(M.confused < drunk_value)
 				M.confused += 3
 		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1)
-	..()
-	return
+			M.adjustToxLoss(1, 0)
+			. = 1
+	return ..() || .
```

Comment:
I think it's when you have datum/reagent/X/Y and both X/on_mob_life() and Y/on_mob_life() can return 0 or 1. If one of the two returns 1 we want to return 1 (because an update is required), we only return 0 if both do not require an update.
When the parent never requires an update the line isn't needed.